Sources/PagecallSDK/NativeBridge.swift

```diff
@@ -121,10 +127,22 @@ class NativeBridge {
 
             print("Bridge Action: \(bridgeAction)")
 
+            if bridgeAction != .`init` && self.chimeController == nil {
+                self.response(requestId: requestId, errorMessage: "ChimeController does not exist")
+                return
+            }
+
             switch bridgeAction {
+            case .`init`:
+                if self.chimeController == nil {
+                    self.chimeController = .init(emitter: self.emitter)
+                    self.response(requestId: requestId)
+                } else {
+                    self.response(requestId: requestId, errorMessage: "ChimeController already exists")
+                }
```

Comment:
[Optional]
[Optional binding](https://velog.io/@wook4506/iOS-Swift-%EC%98%B5%EC%85%94%EB%84%90-%EB%B0%94%EC%9D%B8%EB%94%A9-Optional-Binding) 을 이용해 더 swift 스럽게 짜기 (`?` 를 최소화하여 실수할 여지를 줄입니다)

```suggestion
            if bridgeAction == .`init` {
                if let _ = chimeController {
                    self.response(requestId: requestId, errorMessage: "ChimeController already exists")
                } else {
                    self.chimeController = .init(emitter: self.emitter)
                    self.response(requestId: requestId)
                }
            } else if let chimeController = chimeController {
              switch bridgeAction {
                case .start:
                  chimeController.start { ... }
                ...
              }
            } else {
                self.response(requestId: requestId, errorMessage: "ChimeController does not exist")
            }
```

Reply:
[fix: use optional binding](https://github.com/pagecall/pagecall-ios-sdk/pull/10/commits/193bae9cd4477b88626edb8424c7dc0bd81e67ba)